Description: Grab translatable strings from debian/guest-session-auto.sh
 This is done as an Ubuntu patch, since debian/guest-session-auto.sh
 is not an upstream file.
Forwarded: not-needed
Author: Gunnar Hjalmarsson <gunnarhj@ubuntu.com>

--- lightdm.orig/po/POTFILES.in	2014-01-24 14:44:56.240966000 +0100
+++ lightdm/po/POTFILES.in	2014-01-24 15:09:58.651191195 +0100
@@ -1,5 +1,6 @@
 # List of source files containing translatable strings.
 # Please keep this file sorted alphabetically.
 [encoding: UTF-8]
+debian/guest-session-auto.sh
 src/lightdm.c
 src/dm-tool.c
